tests/unit/versioningconfig_test.py

```diff
@@ -53,3 +57,22 @@ def test_config(self):
         xml.marshal(config)
         self.assertEqual(config.status, SUSPENDED)
         self.assertEqual(config.mfa_delete, DISABLED)
+
+    def test_config_with_excluded_prefixes(self):
+
+        config = xml.unmarshal(
+            VersioningConfig,
+            """
+            <VersioningConfiguration>
+                <Status>Enabled</Status>
+                <MFADelete>Disabled</MFADelete>
+                <ExcludedPrefixes>
+                    <ExcludedPrefix>prefix</ExcludedPrefix>
+                </ExcludedPrefixes>
+            </VersioningConfiguration>
+            """,
+        )
+        xml.marshal(config)
+        self.assertEqual(config.status, ENABLED)
+        self.assertEqual(config.mfa_delete, DISABLED)
+        self.assertListEqual(config.excluded_prefixes, ["prefix"])
```

Comment:
```suggestion
```

Reply:
Why remove the test?

Reply:
We don't need any minio server specific tests here.

Reply:
Not needed as we may change the specific when required.